### ISSUE TITLE
Stocker le login UTC de l'user dans l'user Entity.

### ISSUE THREAD

**Issue by mattgu74** [payutc/onyx#21]
Quand on a un utilisateur de l'UTC, il faut qu'on stocke son login.

**Comment by flo-sch** [comment]
On laisse à JP ça ? Bon petit exo pour voir s'il a compris comment marchaient les entités :)

**Comment by mattgu74** [comment]
@ejpcmac on t'assigne à cette tache, commence par la, tu en auras besoin pour faire l'issue #10

**Comment by mattgu74** [comment]
Le setter et le getter introduit par la PR #26 sont la pour ça.

**Comment by flo-sch** [comment]
On peut fermer cette PR du coup ?

**Comment by mattgu74** [comment]
Non, du peux que j'en ai vu on a introduit le getter et le setter.
Mais lors de la création de l'utilisateur par le CAS, le setter n'est pas appelé donc le login n'est pas "set". De même lors d'une création "traditionnel" on ne check pas si le user à un mail en @utc ou @escom (cas ou il faudrait determiner le login (vu que la tache est compliqué je recommanderait plutôt lors du login traditionnel de détecter les adresses @utc et @escom et d'automatiquement bloquer le process et rediriger vers le CAS))

**Comment by mattgu74** [comment]
Et c'est pas une PR, c'est une issue ^^

**Comment by utc-zz** [comment]
Hello,

I received this email but unfortunately my French is not good enough to 
understand what it's about. Could you explain it, please? Thanks!

Best regards
Toni

**Comment by mattgu74** [comment]
Please forgive us, 
This project is about a ticketing system for students events and we were talking abouts detecting students acount with email extension (in occurence @utc.fr and @escom.fr) unfortunately I didn't think about the fact that there is an user with utc as login (but in fact yes, it's you... )

As you have certainly noticed, this comment absolutely does'nt matter with you.

We are sorry.

Best regards,
Matthieu

**Comment by utc-zz** [comment]
Oh, I see. No problem :)

Best regards
Toni
